server/test/CoreConfigTest.java

```diff
@@ -22,10 +22,10 @@
 import com.vaticle.typedb.core.common.exception.TypeDBException;
 import com.vaticle.typedb.core.server.parameters.util.Option;
 import com.vaticle.typedb.core.server.parameters.util.YamlParser;
-import com.vaticle.typedb.core.server.parameters.Config;
+import com.vaticle.typedb.core.server.parameters.CoreConfig;
```

Comment:
All the changes in this file are from prefixing `Config`, `ConfigFactory` and `ConfigParser` classes with the word `Core`.